redis-ha/templates/tests/test-redis-ha-pod.yaml

```
@@ -0,0 +1,32 @@
+{{- if .Values.haproxy.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-service-test
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+  annotations:
+  {{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 4 }}
+  {{- end }}
+    "helm.sh/hook": test-success
+spec:
+  nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 4 }}
+  tolerations:
+{{ toYaml .Values.tolerations | indent 4 }}
+  containers:
+  - name: "{{ .Release.Name }}-service-test"
+    image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+    command:
+      - sh
+      - -c
+      - redis-cli -h {{ template "redis-ha.fullname" . }}-haproxy -p {{ .Values.redis.port }} info server
```

Is the shell necessary there? Could it not just be
```suggestion
    command:
      - redis-cli
      - "-h"
      - {{ template "redis-ha.fullname" . }}-haproxy
      - "-p"
      - {{ .Values.redis.port }}
      - info
      - server
```